po/zh_CN.po

```diff
@@ -11,7 +11,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Pacemaker 2\n"
 "Report-Msgid-Bugs-To: developers@clusterlabs.org\n"
-"POT-Creation-Date: 2024-02-01 16:24-0800\n"
+"POT-Creation-Date: 2024-02-07 20:47-0800\n"
```

Comment:
don't bother with this commit -- if no actual strings changed, the line numbers aren't important until release time

Reply:
done